src/test/java/org/gridsuite/modification/server/NetworkModificationTest.java

```diff
@@ -90,9 +105,46 @@ public static Network createNetwork() {
         createSwitch(v2, "v2bload", "v2bload", SwitchKind.BREAKER, true, false, false, 4, 5);
         createLoad(v2, "v2load", "v2load", 5, 0., 0.);
 
+        int node1 = 3;
+        int node2 = 6;
+        int node3 = 0;
+        VoltageLevel v3 = createVoltageLevel(s1, "v3", "v3", TopologyKind.NODE_BREAKER, 125.0);
```

Comment:
Why do you need to create all these transformer, tap changer ? You don't use it in your unit test?